package/MDAnalysis/analysis/rdf.py

```diff
@@ -95,6 +95,10 @@
       :class:`numpy.ndarray` of the centers of the `nbins` histogram
       bins.
 
```

Comment:
Please also add in the `new` attributes, see here for an example: https://github.com/MDAnalysis/mdanalysis/blob/da46e8406b863503004daa06686ba86da0428f39/package/MDAnalysis/analysis/rms.py#L120-L133